### PR TITLE
Prepare major

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this are some fixes for the next major release.

## Changelog

```markdown
### Added
- Added missing `swiftmailer` dependency

### Changed
 - Lowered upper composer dependencies

### Removed
 - Removed removed user model properties

### Fixed
 - Fixed PHPDoc
 - Fixed wrong parent calls in UserManager
 - Fixed wrong router call in Controller
```

## Subject

These are some fixes for the next major release.

### CODE REVIEW

**Comment by core23** [comment]
Sorry to create one single PR for a bunch of changes. Feel free to cherry pick if you like.

File: composer.json [hunk 1]
```diff
@@ -55,9 +55,9 @@
         "jms/serializer": "<0.13 || >=2.0",
         "nelmio/api-doc-bundle": "<2.4 || >= 3.0",
         "sonata-project/block-bundle": "<3.2",
-        "sonata-project/doctrine-orm-admin-bundle": "<3.0 || >=4.0",
-        "sonata-project/google-authenticator": "<1.0 || >= 2.0",
-        "sonata-project/seo-bundle": "<2.0 || >=3.0"
+        "sonata-project/doctrine-orm-admin-bundle": "<3.0",
+        "sonata-project/google-authenticator": "<1.0",
+        "sonata-project/seo-bundle": "<2.0"
```
Comment (greg0ire) [inline composer.json]: Did you mean "lower deps"?

File: Controller/AdminResettingController.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ public function sendEmailAction(Request $request)
 
         if (null !== $user && !$user->isPasswordRequestNonExpired($ttl)) {
             if (!$user->isAccountNonLocked()) {
-                return new RedirectResponse($this->router->generate('sonata_user_admin_resetting_request'));
+                return new RedirectResponse($this->get('router')->generate('sonata_user_admin_resetting_request'));
```
Comment (greg0ire) [inline Controller/AdminResettingController.php]: What's wrong with the old way? Does this fix a bug?

File: Model/User.php [hunk 3]
```diff
@@ -177,46 +177,6 @@ public function getUpdatedAt()
...
-    public function setCredentialsExpireAt(\DateTime $date = null)
-    {
-        $this->credentialsExpireAt = $date;
-
-        return $this;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
```
Comment (greg0ire) [inline Model/User.php]: Why this commit?

File: Controller/AdminResettingController.php (hunk 2 above)
Comment (core23) [inline Controller/AdminResettingController.php]: `$this->router` is undefined

File: Model/User.php (hunk 3 above)
Comment (core23) [inline Model/User.php]: This properties were removed in `FOSUserBundle` 2.x

File: composer.json (hunk 1 above)
Comment (core23) [inline composer.json]: No, the upper deps were removed

Comment (greg0ire) [inline composer.json]: And since we are in the conflict section, upper deps are now allowed, got it!

**Comment by jordisala1991** [comment]
Thank you @core23 !

**Pull request merged**